web/app/routes/job.js

```diff
@@ -37,9 +107,40 @@ export default Ember.Route.extend({
         })
       });
     },
+    submitUserChanges(job) {
+      const jobDefId = job.get("jobdefid");
+      const schedulerName = job.get("scheduler");
+      const cluster = job.get("cluster");
+      const cookieName = "elephant." + cluster + ".session.id";
+      const scheduler = new Scheduler();
+      const schedulerUrl = scheduler.getSchedulerUrl(jobDefId, schedulerName);
+      if (!Cookies.get(cookieName)) {
+        this.doLogin(schedulerUrl, cluster);
+      } else {
+        var userAuthorizationStatus = this
+            .getUserAuthorizationStatus(jobDefId, schedulerUrl, cookieName);
+        if (userAuthorizationStatus === "authorised") {
+          //call the param change function
+          //clear error for previus attempt if exists
+          this.clearError();
+        } else if (userAuthorizationStatus === "unauthorised") {
```

Comment:
Create a key value pair for status vs error message. Depending on whateveer userAuthorizationStatus you get, simply return the value using map.get

Reply:
Sure, will do it in future release.